src/components/container/AIAssistant/AIAssistant.tsx

```
@@ -86,10 +87,37 @@ export default function (props: AIAssistantProps) {
+                                }
+
+                                const blob = new Blob([uint8Array], { type: 'image/png' });
+                                const file = new File([blob], `graph-${new Date().getTime()}.png`, { type: 'image/png' });
+
+                                await MyDataHelps.uploadFile(file, "ai-graph");
+                            }
+                        }
+                    }
+                    else if (toolName === "getUploadedFile") {
+                        let input = JSON.parse(streamEvent.data.input.input);
+                        if (input.key && input.key.endsWith(".png")) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)